### ISSUE TITLE
FREEKICK state makes the remain time inconsistent (jumps for and back)

### ISSUE THREAD

**Issue by SGSSGene** [RoboCup-Humanoid-TC/GameController#160]
Setup:
0. using commit #152174d1
1. Adjusted Sender.java:165 by adding a printf:
```
                if (data != null) {
                    data.updateTimes();
                    System.out.println("GC times " + String.valueOf(data.secsRemaining));
...
```
2. started webots with two teams. (didn't start any robot software)
3. waited for play phase
4. moved the robot onto the field and pushed the ball between its legs to force ball holding
5. When a "DIRECT FREEKICK" is given, the line above prints something like:
```
GC times 15
GC times 15
GC times 15
GC times 10
GC times 15
GC times 15
GC times 15
GC times 10
```

I expected a consistent numbers that gets lower, but instead there seem to be some deterministic random numbers inbetween.

**Comment by SGSSGene** [comment]
Ok, the issue with this is that `data.updateTimes()` changes the state of `data`. In most times inbetween two `data.updateTimes()` calls, there is are multiple calls to `Sender::send(AdvancedData)`. In this case everything works fine.
But if two `data.updateTimes()` are being called without a `Sender::send` inbetween the mutated state of `data` will cause wrong times.

**Comment by MaikePaetzel** [comment]
This should be solved with PR #163
